### PR TITLE
Settings Page (Zach) + Navigation Menu (Tim)

### PR DESCRIPTION
**Pull request by Borghese-Gladiator**
INCLUDES:
- Zach's Settings page (pop-up for edit contact info, pop-up for edit profile)
- moved Profile, Settings to /components/private folder
- created PrivateNavbar (navigation menu - appbar && sidebar)

### CODE REVIEW

**Comment by smalhotra3599** [comment]
@Borghese-Gladiator if this is good for review can you add Rishabh or Golden to review this

**Comment by goldenwo** [comment]
Going through the navbar, I found no issues. The linking works perfectly fine, and looking through the code, it seems to be fine too. For the settings page, maybe we don't even need to display the profile picture and change profile picture button. Also add in the username/user id part, separate from the person's actual name. Otherwise everything works perfectly.

File: src/components/Private/Settings.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  };
+
+  render() {
+    const { classes } = this.props;
+    return (
+      <Container component="main" maxWidth="sm">
+        <CssBaseline />
+        <Paper elevation={3} className={classes.paper}>
+          <div className={classes.wrapper}>
+            <Avatar alt="profile photo" src={PlaceholderProfileImg} className={classes.avatar} />
```
Comment (goldenwo) [inline src/components/Private/Settings.tsx:97]: You can remove the profile picture and change profile picture button if you chose to do so.
```suggestion
```

File: src/components/Private/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            <Typography component="h1" variant="h4">
+              John Doe
+            </Typography>
+            <Typography>
+              <Link href="#" className={classes.profilePicLink}>Change profile picture</Link>
```
Comment (goldenwo) [inline src/components/Private/Settings.tsx:102]: You can remove the profile picture and change profile picture button if you chose to do so.
```suggestion
```

File: src/components/Private/Settings.tsx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+            </Typography>
+            <Typography className={classes.statistics}>
+              Email: JDoe@ukg.com
+            </Typography>
+            <Typography>
+              Phone Number: (401)-867-5309
+            </Typography>
+
```
Comment (goldenwo) [inline src/components/Private/Settings.tsx:110]: Add in a part for user id or username
```suggestion
<Typography>
    User ID: 24610
</Typography>
```

**Review by goldenwo — APPROVED** [review]
Looks good, just recommended a couple of optional changes to make for the settings page.

**Pull request merged**